frontend/wallet/src/render/components/Header.re

```diff
@@ -36,17 +36,26 @@ module Styles = {
       justifyContent(`spaceBetween),
     ]);
 
+  let settingsButton =
```

Comment:
is this used somewhere? why pull it out?

Reply:
moved it back in